stable/artifactory-ha/README.md

```diff
@@ -476,6 +476,9 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.image.repository`    | Container image                   | `docker.bintray.io/jfrog/nginx-artifactory-pro`        |
 | `nginx.image.version`       | Container version                 | `.Chart.AppVersion`                                    |
 | `nginx.image.pullPolicy`    | Container pull policy             | `IfNotPresent`                                         |
+| `nginx.logger.image.repository`| repository for logger image | `busybox`                                           |
+| `nginx.logger.image.tag`       | tag for logger image | `1.30`                                                     |
```

Comment:
How about merging this with the `artifactory.logger.image.repository` and `artifactory.logger.image.repository` and giving a common name like `logger.image.repository` and `logger.image.tag` or just using the same as artifactory?

Reply:
What do we do about the different logger names though between artifactory and nginx? Should we have the top level logger only be for the image and then the logger in nginx and artifactory specify the names of the logs?

Reply:
Yes. They should be as you set them until now. Just the image properties should be common.